inc/core/dynamic_css.php

```diff
@@ -99,7 +99,10 @@ public static function minify_css( $css ) {
 	public function add_customize_vars_tag() {
 		wp_register_style( 'nv-css-vars', false );
 		wp_enqueue_style( 'nv-css-vars' );
-		wp_add_inline_style( 'nv-css-vars', self::minify_css(':root{' . $this->get_css_vars() . '}' ) );
+
+		$css = ':root{' . $this->get_css_vars() . '}';
+		$css .= apply_filters( 'neve_after_css_root', $css );
```

Comment:
Why do you apply the same filter twice? here 👆 and here https://github.com/Codeinwp/neve/pull/2783/files#diff-1f14dcbb2fabbdeedd5ee5337cec3c42701a242214f8ee67fb7d9a6c14f8b46bR121

Reply:
Because `$this->get_css_vars()` is called twice once for customizer and once for frontend. Without this it does not reflect the changes inside customizer.